tests/utils/test_modeling_tf_core.py

```diff
@@ -218,6 +218,10 @@ def test_saved_model_creation_extended(self):
             model = model_class(config)
             num_out = len(model(class_inputs_dict))
 
+            for key in list(class_inputs_dict.keys()):
```

Comment:
Do you need to cast to a list here? 
```suggestion
            for key in class_inputs_dict:
```

Reply:
I assumed I needed to because otherwise I'd be modifying the dict as I'm iterating over it and I know that can cause problems with Python, though possibly it still works in this situation because the keys stay the same.

Reply:
I'm pretty sure it's OK as it's just the values being modified (not the keys or number of entries). It does feel like a python trap though 😅

Reply:
Done!